src/NextjsBuild.ts

```diff
@@ -55,6 +55,12 @@ export class NextjsBuild extends Construct {
 
   public nextDir: string;
   public projectRoot: string;
+  /**
+   * The pure nextjs path, eg excluding "../" from "../../web"
+   * This is required when building Lambda handler paths, since its context
+   * is isolated to only its nextjs directory
+   */
+  public pureNextJsPath: string;
```

Comment:
Reading this as a user I have no clue what it is or why/when I should set it

Reply:
Very true, I suck at naming things. Any suggestions?

Reply:
This may be better as an internal variable if it isn't going to be used elsewhere.